spec/requests/providers/check_provider_answers_spec.rb

```diff
@@ -209,8 +209,8 @@
         end
 
         context 'non passported' do
-          it 'redirects to the check benefits page' do
-            expect(response).to redirect_to(providers_legal_aid_application_check_benefits_path(application))
+          it 'redirects to the confirm dwp non passported applications page' do
+            expect(response).to redirect_to(providers_legal_aid_application_confirm_dwp_non_passported_applications_path(application))
```

Comment:
I don't think the changes to this file are needed. This is reverting changes that were made for the DWP Override functionalilty and are needed on master.